### DISCUSSION TITLE
Fix needed for deprecated function escape() vulnerability which is found in Bootstrap v3.3.7

### DISCUSSION THREAD

**Discussion by imshesh** [twbs/bootstrap#39821] [Q&A]
Hello,  I have .Net core MVC application which uses Bootstrap V3.3.7 , JQuery v3.7.1 and JQueryUI V1.13.2 along with Kendo.
I have checkmarx vulnerability to fix which says, HRA JAVASCRIPT Find Deprecated Functions i.e. this.escape().
Which bootstrap version has fix for this?. If upgrade is needed which is closest version to upgrade to minimize breaking changes.

Thanks for the support

**Comment by julien-deramond** [comment]
> Which bootstrap version has fix for this?. If upgrade is needed which is closest version to upgrade to minimize braking changes.

IDK exactly what Bootstrap version would have a fix for that, versions 3 are pretty old and the message is not really verbose (without the associated code).

Anyway, maybe you can try an incremental approach and bump Bootstrap dependency step-by-step until the error disappears. The next versions of 3.x after 3.3.7 are [3.4.0](https://github.com/twbs/bootstrap/releases/tag/v3.4.0) and [3.4.1](https://github.com/twbs/bootstrap/releases/tag/v3.4.1) (that fixed some vulnerabilities issues).

If it's linked to the `this.escape()` function in Bootstrap code, it disappears from the version v4.0.0. But the breaking changes are more important.
